src/commands/kol/offers.ts

```diff
@@ -92,6 +92,7 @@ async function viewStandingOffers(interaction: ChatInputCommandInteraction) {
 
   const embed = createEmbed()
```

Comment:
I think you need an `await` here as well. That or you should slice out the `await wikiClient.getWikiLink(item)` into a variable above the embed.

Reply:
Probably the second one

Reply:
No this is fine